### ISSUE TITLE
Leaving SKE taints empty will result in an error

### ISSUE THREAD

**Issue by h3adex** [stackitcloud/terraform-provider-stackit#456]
Leaving SKE taints empty will result in an error.

Code:
```tf
resource "stackit_ske_cluster" "ske_cluster_01" {
  project_id             = var.project_id
  name                   = var.cluster_name
  kubernetes_version_min = "1.29.6"
  node_pools = [
    {
      name               = "gpu-pool-1"
      machine_type       = ""
      os_min_version     = ""
      os_name            = "ubuntu"
      minimum            = "1"
      maximum            = "1"
      availability_zones = ["eu01-3"]
      volume_size        = 20
      volume_type        = "storage_premium_perf1"
      taints = []
    }
  ]
}
```

![Screenshot 2024-07-09 at 14 24 26](https://github.com/stackitcloud/terraform-provider-stackit/assets/39736813/fd75c70a-1f6b-4f21-b8af-89f1c479337b)

Expected behavior:
node-pool has no taints

**Comment by DiogoFerrao** [comment]
Hey @h3adex, thank you for reporting this!

I will be taking a look at this and provide you with an update soon.

Kind regards,
Diogo

**Comment by joaopalet** [comment]
Hi @h3adex,

This was addressed and fixed in [`v0.24.1`](https://registry.terraform.io/providers/stackitcloud/stackit/latest)
